### PR TITLE
Readme Extend installation and usage 

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,37 +32,51 @@ docker run -i -t -p 8080:80 backofenlab/docker-galaxy-graphclust
...
+## Browser access to the server:
+After running the Galaxy server, a web server is established under the host IP/URL and designated port.
+
+* Inside your browser goto IP/URL:PORT
+* Following same settings as previous step: 
+    * In the **same local computer**: [http://localhost:8080/](http://localhost:8080/)
+    * In **any computer with network connection to the host**: [http://HOSTIP:8080/]()
+    
+## Registration and Login: 
+To have distinct history and workflows the Galaxy server requires each user to register for first access time. **By default anyone with access to the host network can register. No registration confirmation email will be sent to the given email.** So you can register with any custom (including non-existent) email address. To change the default authorization settings please refer to Galaxy Wiki [Authentication section](https://wiki.galaxyproject.org/Develop/Authentication)
```
Comment (mmiladi) [inline README.md]: @bgruening Is the authentication description correct? Is no email sent statement true?

Comment (bgruening) [reply README.md]: Yes, but we can also add a sentence that there is a default Admin user. For more information I would always link out to the original Galaxy-Docker documentation: http://bgruening.github.io/docker-galaxy-stable/users-passwords.html

File: README.md [hunk 2]
```diff
@@ -2,12 +2,17 @@
 [![Build Status](https://travis-ci.org/BackofenLab/docker-galaxy-graphclust.svg?branch=master)](https://travis-ci.org/BackofenLab/docker-galaxy-graphclust)
 [![Docker Repository on Quay](https://quay.io/repository/bgruening/galaxy-graphclust/status "Docker Repository on Quay")](https://quay.io/repository/bgruening/galaxy-graphclust)
 
-Galaxy GraphClust Flavor
+Galaxy-GraphClust
 ========================
+Galaxy-GraphClust is a workflow for structural clustering of RNA secondary structures developed as an extension of GraphClust clustering pipeline inside the Galaxy framework. It consists of a set of integrated Galaxy tools and different flavors of clustering workflows built upon these tools.
```
Comment (bgruening) [inline README.md]: `GraphClust clustering pipeline` should we cite here the original publication?

File: README.md (hunk 1 above)
Comment (bgruening) [reply README.md]: Yes, no email is send out until you configure it.

File: README.md [hunk 3]
```diff
@@ -27,37 +32,51 @@ docker run -i -t -p 8080:80 backofenlab/docker-galaxy-graphclust
...
+* Following same settings as previous step: 
+    * In the **same local computer**: [http://localhost:8080/](http://localhost:8080/)
+    * In **any computer with network connection to the host**: [http://HOSTIP:8080/]()
+    
+## Registration and Login: 
+To have distinct history and workflows the Galaxy server requires each user to register for first access time. **By default anyone with access to the host network can register. No registration confirmation email will be sent to the given email.** So you can register with any custom (including non-existent) email address. To change the default authorization settings please refer to Galaxy Wiki [Authentication section](https://wiki.galaxyproject.org/Develop/Authentication) 
 
-# Usage:
+* To register (first time only):
+    * On top right of the panel goto **User->Register**
```
Comment (bgruening) [inline README.md]: `->` - →

File: README.md [hunk 4]
```diff
@@ -27,37 +32,51 @@ docker run -i -t -p 8080:80 backofenlab/docker-galaxy-graphclust
...
-## To launch the Galaxy GraphClust pipeline:
+* To login:
+    * On top right of the panel goto **User->Login**
+    * Provide your registered email address and password
 
-### From command line:
-* Inside your browser goto [http://localhost:8080/](http://localhost:8080/)
+## Help
+### Interactive tours
+Interactive Tours are available for Galaxy and Galaxy-GraphClust. To run the tours please on top panel go to **Help->Interactive Tours** and click on one of the tours prefixed **GraphClust workflow**. You can check the other tours for a more general introduction to the Galaxy interface.
```
Comment (bgruening) [inline README.md]: We should start this specific tour from the front-page as well.

File: README.md (hunk 2 above)
Comment (mmiladi) [reply README.md]: Certainly !

File: README.md (hunk 4 above)
Comment (mmiladi) [reply README.md]: You mean to just put the link inside welcome.html or automatically trigger it?

**Comment by mmiladi** [comment]
I did kinda mess by accidentally merging open branches :-D It should be no problem in final merges with master.

**Comment by bgruening** [comment]
Everything looks good :)

**Pull request merged**